examples/fancy-data-types.ts

```diff
@@ -1,17 +1,25 @@
 import * as tucson from "../";
+import {Decoder} from "../lib";
```

Comment:
Use `tuscon.Decoder` instead so the example shows that the interface is available from the import declaration in the line above.